modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/data/provider/AnalyticsReportsDataProvider.java

```diff
@@ -47,6 +54,51 @@ public AnalyticsReportsDataProvider(Http http) {
 		_asahFaroBackendClient = new AsahFaroBackendClient(http);
 	}
 
+	public Map<String, AcquisitionChannel> getAcquisitionChannels(
+			long companyId, String url)
+		throws PortalException {
+
+		try {
+			String response = _asahFaroBackendClient.doGet(
+				companyId,
+				"api/1.0/pages/acquisition-channels?canonicalURL=" + url +
```

Comment:
interval and rakeKey are fixed because we just want to replace the current behavior, right?

Reply:
These params are fixed because the story says so.

Reply:
In the future maybe a selector for the interval could be designed but we should deal with the difference between the info of the traffic source pie chart and the SEMRush endpoint that provides the keywords.

Reply:
👌  just wanted to confirm